### ISSUE TITLE
rados driver does not create root directory

### ISSUE THREAD

**Issue by stevvooe** [docker/distribution#832]
As part of #831, a new test has been added to ensure that a root directory is created. Without this, there is no way to implement a common backend health check.

Rados fails this check:

```
----------------------------------------------------------------------
FAIL: /home/ubuntu/.gvm/pkgsets/stable/global/src/github.com/docker/distribution/registry/storage/driver/testsuites/testsuites.go:91: DriverSuite.TestRootExists

/home/ubuntu/.gvm/pkgsets/stable/global/src/github.com/docker/distribution/registry/storage/driver/testsuites/testsuites.go:94:
    c.Fatalf(`the root path "/" should always exist: %v`, err)
... Error: the root path "/" should always exist: rados: ret=-5

OOPS: 31 passed, 2 skipped, 1 FAILED
--- FAIL: Test (59.22s)
FAIL
coverage: 78.3% of statements
FAIL	github.com/docker/distribution/registry/storage/driver/rados	59.261s
```

@lebauce @nevermosby

**Comment by dmp42** [comment]
Wasn't rados on @gierschv ?

**Comment by stevvooe** [comment]
O no. I got the swift guys... we should write this down somewhere.

cc @simon3z @noahdesu @gierschv

**Comment by noahdesu** [comment]
Hmm, that -5 return value looks like -EIO. A backtrace would be useful, but the most efficient would be to reproduce this locally. I've supported the `go-ceph` side of this, but never ran the Docker side. Would someone mind pointing me at a document or instructions for settings things up and running this?

**Comment by dmp42** [comment]
@noahdesu can you join irc on #docker-distribution ?

**Comment by noahdesu** [comment]
Unfortunately I am not in a position to get on IRC today.

**Comment by RichardScothern** [comment]
ping @simon3z @noahdesu @gierschv

We cannot enable the Rados storage driver with the 2.1 release with this problem outstanding.

**Comment by gierschv** [comment]
I'm checking this issue right now

**Comment by gierschv** [comment]
PR #845

**Comment by dmp42** [comment]
Should be fixed by #845
